packages/date/package.json

```diff
@@ -32,13 +32,15 @@
   },
   "peerDependencies": {
     "@availity/form": "^0.1.0",
+    "@availity/yup": "^2.0.0",
```

Comment:
This is a breaking change and will need to be noted to major bump `@availity/date`